README.md

```
@@ -64,6 +64,17 @@
 pnpm lighthouse:mobile - lighthouse mobile tesitng
 ```
 
+## Routing
+
+This project includes a routing script for managing URLs. The routing script ensures that requests to various paths are mapped to their corresponding HTML files. For detailed information, check the [routing script](scripts/cloudfront_routing.js).
+How It Works:
+
+  - Mapping: Specific URL paths are mapped to corresponding HTML files.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)